server/package.json

```diff
@@ -11,7 +11,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "uuid": "^9.0.0"
```

Comment:
Since you are not using this package, it's better to remove it from your `package.json` file

Reply:
I have uninstalled the uuid package from my project